cmd/clusterawsadm/cmd/root.go

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2018 The Kubernetes Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"sigs.k8s.io/cluster-api-provider-aws/cmd/clusterawsadm/cmd/alpha"
+)
+
+var RootCmd = &cobra.Command{
```

Comment:
package level vars can be hard to work with and easily abused. It's ok to get get this moving right now, but can we track this as cleanup work in an issue if you agree?

Reply:
Yup. This is straight up hackiness based on copying directly from the cluster-api repo.